src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-howto-retain-multiple-blobs.md

```diff
@@ -0,0 +1,240 @@
+review:
+    comment: ''
+    date: '2024-01-22'
+    status: 'ok'
+labels:
+    - bchauvin
+    - mlumeau
+    - retention-management
+toc: true
+tree_item_index: 450
+---
+
+Proving compliance requires keeping a record of all the evidence you may need. Therefore, Nuxeo Server allows you to define in the most flexible way what should constitute a record. 
+
+By default, Nuxeo Server will always protect the main file of the document when [putting a document under retention]({{page page='nuxeo-retention-functional-overview'}}#put-a-document-under-retention) and the document remains editable. Should you wish to put under retention additional files from the document and prevent certain properties from being edited, you need to configure a list of properties to be protected.
+
+In this tutorial, you will learn how to leverage this configuration to retain multiple document properties securely, including how to retain additional files in WORM storage. This helps proving that no metadata has been tampered with in case of an audit. If you wish to go further, a high level understanding of how to store additional elements like comments will also be provided.
+
+## Prerequisites
+
+This feature is available starting from:
+- Nuxeo Server `LTS 2021 HF33`
+- Nuxeo Retention `2021.4.5`
+
+This how-to requires:
+- Being familiar with the base concepts of Nuxeo and Nuxeo Studio.
+- Knowing how  to use Nuxeo Web UI for standard actions like navigating and creating a document.
+
+## Define Properties to Retain 
+
+Properties to retain are defined through Nuxeo Studio configuration.
+
+First, we need to declare the dependency on the addon.
+
+In Nuxeo Studio Modeler:
+- Go to the `Application Definition` screen, and add `Nuxeo Retention` to the list of packages to install.
+- Save your configuration
+
+Now, let's configure a custom document type containing additional properties we want to retain.
+
+In Nuxeo Studio Modeler:
+- Go to `Content Model > Document Types`
+- Create a new document type
+  - Name it `MyClaim`
+  - Make it extend `File`
+- In the `Schema` tab, add the following custom properties:
+  - `amountRefunded`: floating point
+  - `expertReport`: blob
+  - `incidentDate`: date
+  - `incidentDescription`: string
+  - `repairQuotes`: blob (multi-valued)
+
+We need to display these properties during creation, edit and view. 
+
+- Go to Nuxeo Studio Designer
+- Go to `Layouts > Local Document Types > MyClaim` and click on the `Configure Missing Layouts` button.
+- In the different layouts, drag and drop the `myclaim` schema to add the missing properties to the form and save your configuration.
+
+Next step is to configure which of these properties we want to retain.
+
+In Nuxeo Studio Modeler:
+- Go to `Advanced Settings > XML Extensions`
+- Create a new XML extension and give it a name, e.g. `retainMyClaimProperties`
+- Paste the following code
+
+```
+<require>org.nuxeo.ecm.core.CoreExtensions</require> 
```

Comment:
indentation

Reply:
Sorry, not sure what's wrong with indentation here, do you mind providing a suggestion?

Reply:
```suggestion
  <require>org.nuxeo.ecm.core.CoreExtensions</require>
```